Realm/Realm/Handles/ListHandle.cs

```diff
@@ -54,9 +54,27 @@ private static class NativeMethods
 
             [DllImport(InteropConfig.DLL_NAME, EntryPoint = "list_move", CallingConvention = CallingConvention.Cdecl)]
             public static extern IntPtr move(ListHandle listHandle, ObjectHandle objectHandle, IntPtr targetIndex, out NativeException ex);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "list_get_is_valid", CallingConvention = CallingConvention.Cdecl)]
+            [return: MarshalAs(UnmanagedType.I1)]
+            public static extern bool get_is_valid(ListHandle listHandle, out NativeException ex);
+
+            [DllImport(InteropConfig.DLL_NAME, EntryPoint = "list_get_thread_safe_reference", CallingConvention = CallingConvention.Cdecl)]
+            public static extern ThreadSafeReferenceHandle get_thread_safe_reference(ListHandle listHandle, out NativeException ex);
+        }
+
+        public override bool IsValid
+        {
+            get
+            {
+                NativeException nativeException;
+                var result = NativeMethods.get_is_valid(this, out nativeException);
+                nativeException.ThrowIfNecessary();
+                return result;
+            }
         }
 
-        internal ListHandle(RealmHandle root) : base(root)
+        public ListHandle(RealmHandle root) : base(root)
```

Comment:
Why is this now public? If it is satisfying a public interface it should have a comment.